### PR TITLE
Merge preview2 to rel100

### PR DESCRIPTION
**Pull request by livarcocc**
cc @eerhardt @brthor @Sridhar-MS @piotrpMSFT

### CODE REVIEW

File: test/dotnet-migrate.Tests/GivenThatIWantToMigrateTestApps.cs [hunk 1]
```diff
@@ -121,6 +121,7 @@ public void It_migrates_an_app_with_scripts_and_the_scripts_run()
         private MigratedBuildComparisonData GetDotnetNewComparisonData(string projectDirectory, string dotnetNewType)
         {
             DotnetNew(projectDirectory, dotnetNewType);
+            File.Copy("NuGet.tempaspnetpatch.config", Path.Combine(projectDirectory, "NuGet.Config"));
```
Comment (eerhardt) [inline test/dotnet-migrate.Tests/GivenThatIWantToMigrateTestApps.cs]: Do we need to do this for every new project?  or just for web projects?

Comment (livarcocc) [inline test/dotnet-migrate.Tests/GivenThatIWantToMigrateTestApps.cs]: Just the web. I can move it.

File: test/dotnet-migrate.Tests/NuGet.tempaspnetpatch.config [hunk 2]
```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+  <packageSources>
+    <!--To inherit the global NuGet package sources remove the <clear/> line below -->
+    <clear />
+    <add key="dotnet-core" value="https://dotnet.myget.org/F/dotnet-core/api/v3/index.json" />
```
Comment (livarcocc) [inline test/dotnet-migrate.Tests/NuGet.tempaspnetpatch.config]: I just copied the one we have in the CLI root.

File: test/dotnet-migrate.Tests/GivenThatIWantToMigrateTestApps.cs (hunk 1 above)
Comment (livarcocc) [inline test/dotnet-migrate.Tests/GivenThatIWantToMigrateTestApps.cs]: Actually, this needs to happen between the new and the restore. So, this is the best place to do it. Unless we have a reason to move it out. I would leave it here.

Comment (brthor) [inline test/dotnet-migrate.Tests/GivenThatIWantToMigrateTestApps.cs]: Im surprised this doesn't just use the nuget.config from the repo root by default. LGTM though

**Comment by brthor** [comment]
:shipit:

**Pull request merged**